### DISCUSSION TITLE
installation instruction is too vague, don't know how to install

### DISCUSSION THREAD

**Discussion by ghost** [Dadoum/Sideloader#43] [Q&A] [answered]
> Currently, the only builds available can be downloaded through GitHub Actions.

what is the meaning of this? I click on the actions page but not surewhat to do. I am not familliar with GitHub's soplease excuse me.

**Comment by Dadoum** [comment]
I should update that section in the Readme. You can go in the release tab directly and download the right executable there for your system.

**Reply by ghost** [reply]
release tab.? not sure what you mean.

**Reply by Dadoum** [reply, marked as answer]
https://github.com/Dadoum/Sideloader/releases

**Reply by ghost** [reply]
of course, thank you.

**Reply by Dadoum** [reply]
np

**Reply by xipe-totec-gnu** [reply]
What's the difference between the normal file and the dbg file?

**Reply by Dadoum** [reply]
The debug file is useful for developers to debug Sideloader crashes.
